Makefile

```diff
@@ -26,3 +26,6 @@ build:
 
 install:
 	go install ${BUILD_FLAGS} ./cmd/secrethub
+
+msi:
```

Comment:
Consider adding the `make msi` command to the goreleaser config as a pre or post build hook (not sure which would be preferred here, I think post). Then we can include the resulting `.msi` files in the release.

Reply:
I tried that at first, but those hooks very limited. Can have another look later.